packages/emails/src/templates/DisabledAppEmail.tsx

```diff
@@ -0,0 +1,86 @@
+import { TFunction } from "next-i18next";
+import { Trans } from "react-i18next";
+
+import { CAL_URL } from "@calcom/lib/constants";
+
+import { BaseEmailHtml, CallToAction } from "../components";
+
+export const DisabledAppEmail = (
+  props: {
+    appName: string;
+    appType: string[];
+    t: TFunction;
+    title?: string;
+    eventTypeId?: number;
+  } & Partial<React.ComponentProps<typeof BaseEmailHtml>>
+) => {
+  const { title, appName, eventTypeId, t, appType } = props;
+
+  return (
+    <BaseEmailHtml subject={t("app_disabled", { appName: appName })}>
+      {appType.some((type) => type === "payment") ? (
```

Comment:
NIT 

This is one big ternary - I wonder if there is a cleaner way of doing this

Let me know if you want me to have a go in a future PR

Reply:
It's either this way or a large if else block. I'm not sure if there's a cleaner way to write this.